<p>
Typically, backslashes are seen only as part of escape sequences. 
Therefore, the use of a backslash outside of a raw string or escape sequence looks suspiciously like a broken escape sequence.
</p>

<p>
Characters recognized as escape-able are: abfnrtvxnNrtuU\\'\"0123456789
</p>

<h2>Noncompliant Code Example</h2>

<pre>
s = "Hello \world."
t = "Nice to \ meet you"
u = "Let's have \ lunch"
</pre>

<h2>Compliant Solution</h2>

<pre>
s = "Hello world."
t = "Nice to \\ meet you"
u = r"Let's have \ lunch"
</pre>